src/purifier.js

```diff
@@ -481,6 +486,14 @@ export function addPurifyHooks(purifier, diffing) {
   purifier.addHook('afterSanitizeAttributes', afterSanitizeAttributes);
 }
 
+/**
+ * @param {string} attrName
+ * @return {boolean}
+ */
+export function isBindingAttribute(attrName) {
+  return attrName[0] == '[' && attrName[attrName.length - 1] == ']';
```

Comment:
nit parenthesis around conditions to improve readability

 (attrName[0] === '[') && (attrName[attrName.length - 1] === ']');